extends Node
var hotel_intro_ended:bool = false
var add_bed:bool = false
var add_bookshelf:bool = false
var add_sign: bool = false
var money = int(0)
var tutorial:bool = true
var bought_bed:bool = false
var bought_bookshelf: bool = false
var bought_sign: bool = false
var advertisement = int(0)
var customers = int(0)
var transition_finish := false
var transition2_finish := false
var hide_tutorial := false
var room1_full := false
var room2_full := false
var room3_full := false
var room4_full := false
var room5_full := false
var room6_full := false
var played_tutorial := false
var review = 0
var checkoutside := false
var advertisement_multiplier := 1
func _process(_float) -> void:
	if checkoutside:
		get_tree().change_scene_to_file("res://hotel_intro.tscn")
func to_hotel_outside():
	get_tree().change_scene_to_file("res://hotel_outside.tscn")
